templates/application-metacontroller-crds.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+metadata:
+  name: metacontroller-crds
```

Comment:
This is just the CRDs? I think it's installing the entire chart.

Reply:
ya good point - I chose a bad name
I'll update it to "application-metacontroller" and "metacontroller"